chapters/5-processing.tex

```diff
@@ -255,7 +254,11 @@ \subsection{Tidying raw data}
 (in Stata, often through \texttt{collapse})
 and then merging or joining those results to the firms data table.
 
-In a tidy workflow, your clean dataset is a set of one or more tidy data tables.
+In a tidy workflow, your clean dataset comes from 
+a group of one or more tidy data tables.
+This is reflected in the data flowcharts from your data map
+showing the way to combine data from the master datasets
+any any other datasets that you obtained.\index{data flowchart}\index{data map}
```

Comment:
I can be reflected but it does not have to be. I think this usage of data flowchart is not the best.  I think this hunk can be reverted